content/applications/finance/accounting/others/adviser/budget.rst

```diff
@@ -2,156 +2,88 @@
+Analytic Accounts` and :guilabel:`Create` a new :guilabel:`Analytic Account` called Smith & Co.
 
-To set those targets, enter the accounting app, select :menuselection:`Advisers -->
-Budgets` and create a new Budget.
+Each budgetary position can have any number of accounts from the general ledger (the main chart of
+accounts) assigned to it, though it must have at least one.
 
-We have to give a name to the budget. In this case we'll call it "Smith
-Project". Select the period wherein the budget will be applicable. Next
-add an item to specify your targets in the Budget Line.
+Define the budget
+-----------------
 
-.. image:: budget/budget07.png
-   :align: center
+Let’s now set our targets. We specified that we expect to gain 1000 with this project and we would
+like not to spend more than 700. Go to :menuselection:`Accounting --> Budgets` and
+:guilabel:`Create` a new budget.
 
-Select the Budgetary Position related to the Budget Line. In other
-words, select the position that points to the accounts you want to
-budget. In this case we will start with our 700 maximum charge target.
-Select the "Cost" Budgetary Position and specify the Planned Amount.
-As we are recording a cost, we need to specify a **negative amount**.
-Finally, select the corresponding analytic account.
+First, fill in your :guilabel:`Budget Name`. Then, select the :guilabel:`Period` wherein the budget
+is applicable. Next, add the :guilabel:`Budgetary Position` you want to track, define the related
+:guilabel:`Analytic Account`, and add the :guilabel:`Planned Amount`.
 
-.. image:: budget/budget08.png
+.. image:: budget/define-the-budget.png
    :align: center
 
-Click on **Save & new** to input the revenue budget. The Budgetary
-Position is Revenue and the Planned Amount is 1000. Save and close
-
-You'll need to **Confirm** and **Approve** the budget.
+.. Note::
+   When recording a planned amount related to expenses, the amount must be negative.
 
 Check your budget
-=================
-
-You can check your budget at any time. To see the evolution, let's book
-some Invoices and Vendors Bills.
-
-.. tip::
-   If you use analytical accounts remember that you need to specify the account in the invoice
-   and/or purchase line.
+-----------------
 
-.. seealso::
-   For more information about booking invoices and purchase orders see
-   :doc:`../../receivables/customer_invoices/overview`
+Go to :menuselection:`Accounting --> Budgets` and find the Smith & CO Project to see how it evolves.
 
-Go back in the budget list and find the Smith Project.
-
-Via the analytical account, Odoo can account the invoice lines and
-purchase lines booked in the accounts and will display them in the
-**Practical Amount** column.
-
-.. image:: budget/budget09.png
-   :align: center
+.. tip:: If you use analytical accounts remember that you need to specify this analytic account in
+   the invoice and/or purchase line.
```

Comment:
This shouldn't be a tip as nothing happens if you don't do it - not a tip but a mandatory step.
   I think you should explain where to add you "analytical accounts"